PoGo.NecroBot.CLI/Config/Translations/translation.id.json

```diff
@@ -334,19 +334,19 @@
     },
     {
       "Key": "maxItemsCombinedOverMaxItemStorage",
-      "Value": "[Konfigurasi Salah] Total barang maksimum kamu (ball+potion+revive={0}) melebihi kapasitas tas kamu ({1})"
+      "Value": "[Konfigurasi Salah] Total barang maksimum Anda (ball+potion+revive={0}) melebihi kapasitas tas Anda ({1})"
     },
     {
       "Key": "recyclingQuietly",
-	  "Value": "Mendaur ulang senyap..."
+	  "Value": "Daur ulang dengan santai..."
```

Comment:
Hello, @awaaas 
In this line, I saw the words were a little stiff. I intend to change the wording so as not to stiffen. Sorry if I am wrong. ^^